src/main/java/edu/greenblitz/robotName/commands/lifter/LifterCommand.java

```diff
@@ -0,0 +1,13 @@
+package edu.greenblitz.robotName.commands.lifter;
+
+import edu.greenblitz.robotName.subsystems.Lifter.Lifter;
+import edu.greenblitz.robotName.utils.GBCommand;
+
+public abstract class LifterCommand extends GBCommand {
+    protected Lifter lifter;
```

Comment:
make private

Reply:
Cannot work because in that case the lifter attribute won't be identified in the "ReverseLifting" and "LiftUp" commsnds.

Reply:
Also, it's written the same way in Charged Up